scripts/publish-release.sh

```diff
@@ -28,7 +28,7 @@ release_target=builtin-actors-${BUILD_FIL_NETWORK}.car
 release_target_hash=builtin-actors-${BUILD_FIL_NETWORK}.sha256
 release_file=output/$release_target
 release_file_hash=output/$release_target_hash
-release_tag="${GITHUB_SHA:0:16}"
+release_tag="${GITHUB_TAG}"
```

Comment:
Where is `GITHUB_TAG` coming from? Is it a well-known env var when creating a manual release?

Reply:
see https://docs.github.com/en/github-ae@latest/graphql/reference/objects#release

Reply:
I set it in the workflow.

Reply:
see also https://docs.github.com/en/github-ae@latest/actions/using-workflows/events-that-trigger-workflows#release

Reply:
See also https://stackoverflow.com/questions/71083657/trigger-github-workflow-only-on-the-release-tag-event-and-specific-path

Reply:
I pused some updates, and now it is set like this:

Reply:
maybe we should use `...release.tagName` though? I don't know, we'll have to test.

Reply:
used GITHUB_REF and extracted the tag from it; it should work now.